### PR TITLE
[KOGITO-5944] Custom json dashboards

### PR DESCRIPTION
**Pull request by gitgabrio**
@danielezonca @r00ta  @jiripetrlik

See https://issues.redhat.com/browse/KOGITO-5944

Depends on https://github.com/kiegroup/kogito-runtimes/pull/1758

### CODE REVIEW

**Review by danielezonca — APPROVED** [review]

**Review by danielezonca — COMMENTED** [review]
Can you please update `dmn-drools-quarkus-metrics` and `dmn-drools-springboot-metrics` README to describe how to add a custom dashboard?
Link https://grafana.com/docs/grafana/latest/dashboards/json-model/ as reference for the json

**Comment by gitgabrio** [comment]
> Can you please update `dmn-drools-quarkus-metrics` and `dmn-drools-springboot-metrics` README to describe how to add a custom dashboard? Link https://grafana.com/docs/grafana/latest/dashboards/json-model/ as reference for the json

@danielezonca Done
See also https://issues.redhat.com/browse/KOGITO-6380

**Review by r00ta — APPROVED** [review]

**Comment by gitgabrio** [comment]
@danielezonca 
Could you pls merge this ? Thanks

File: dmn-drools-quarkus-metrics/src/test/java/org/kie/kogito/examples/quarkus/DashboardsListTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
```
Comment (jiripetrlik) [inline dmn-drools-quarkus-metrics/src/test/java/org/kie/kogito/examples/quarkus/DashboardsListTest.java]: Please use 2021.

File: dmn-drools-quarkus-metrics/src/test/java/org/kie/kogito/examples/quarkus/DashboardsListTest.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+@QuarkusTest
+public class DashboardsListTest {
+
+    @Test
+    @SuppressWarnings("unchecked")
+    public void testDashboardsListIsAvailable() {
+        List<String> dashboards = given().contentType(ContentType.JSON).accept(ContentType.JSON).when()
+                .get("/monitoring/dashboards/list.json").as(List.class);
+
+        Assertions.assertEquals(6, dashboards.size());
```
Comment (jiripetrlik) [inline dmn-drools-quarkus-metrics/src/test/java/org/kie/kogito/examples/quarkus/DashboardsListTest.java:37]: Can we check dashboard names or something?

**Review by jiripetrlik — APPROVED** [review]
I have two minor comments. Otherwise looks good to me.

**Pull request merged**